source/_patterns/04-pages/article--research.mustache

```diff
@@ -60,11 +60,7 @@
             {{! May need to make the article section pattern more content flexible (include other patterns/not have header). New variants? }}
             {{#articleMeta}}
               <section class="article-section">
-
-                <div class="article-section__body">
```

Comment:
Was this change intentional? If so, does this also need to be made to any of the other pages examples to keep them in sync or not?

Reply:
Yes it'sdone intentionally as it's not on production and if this class is in place you do not get the blue links